shell.nix

```diff
@@ -51,8 +47,7 @@ mkShell {
     python3
     utillinux
     xfsprogs
```

Comment:
and these two?

Reply:
right, I also dropped `e2fsprogs`.

Reply:
As said on the slack, I re-added `utillinux`. I also updated the commit log with info on that.